### ISSUE TITLE
CSSLint should support shadow dom component styling 

### ISSUE THREAD

**Issue by uditbh** [CSSLint/csslint#767]
Code like the following results in an 'Expected RPAREN' error:

`:host(.studentScreenSharing[data-currapp='ScreenShare']) span.icon-stdscreenImg {
}`

**Comment by smeans** [comment]
Yeah, I'm running across this now because csslint is integrated into the Ace editor. I've started poking around in the code and am going to try to fix it, but any advice from a committer would be appreciated :)
